Services/Mail/classes/class.ilFormatMail.php

```diff
@@ -31,15 +31,15 @@ public function __construct($a_user_id)
     * @access	public
     * @return string
     */
-    public function formatReplyMessage()
+    public function formatReplyMessage() : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.